src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -42,7 +42,7 @@ public void RelativeSearchPath_Is_Null()
         public void TargetFrameworkTest()
         {
             // On Uap we use the Microsoft.DotNet.XUnitRunnerUap instead of the RemoteExecutorConsoleApp
-            string targetFrameworkName = PlatformDetection.IsUap ? ".NETCore,Version=v5.0" : "DUMMY-TFA";
+            string targetFrameworkName = PlatformDetection.IsUap ? ".NETCoreApp,Version=v2.0" : "DUMMY-TFA";
```

Comment:
@ViktorHofer expected?

Reply:
Seems regular UAP sounds remain the same .NETCore,Version=v5.0

Reply:
Expected yes as we changed the runner. Uap's TFM here is .NETCore,Version=v5.0. With. NET Native (ie uapaot & netcoreapp at) the TFM is .NETCoreApp,Version=v2.0. Everything else is Dummy-TFA.